docs/src/General/Contributing_to_Bazzite.md

```diff
@@ -29,10 +29,9 @@ Check out the Bazzite's [issue tracker](https://github.com/ublue-os/bazzite/issu
 
 # Building Your Own Bazzite for Testing
 
-https://universal-blue.discourse.group/docs?topic=43
+>**[Bazzite Template](https://github.com/ublue-os/image-template)**
 
-**Template**:
-https://github.com/ublue-os/image-template
```

Comment:
@nicknamenamenick you have two links for the same github page one after the other with this change